cleanup_utils/integration_cleanup.py

```diff
@@ -31,7 +31,7 @@
 from pytz import UTC
 
 from cleanup_utils.aws_cleanup import AwsCleanupInstance
-from cleanup_utils.dns_cleanup import DnsCleanupInstance
+from cleanup_utils.dns_cleanup import DNSCleanupInstance
```

Comment:
Nit: `CamelCaseNaming` should always be 1 uppercase + lowercase characters (

Reply:
> Nit: CamelCaseNaming should always be 1 uppercase + lowercase characters 

@giovannicimolin, not when a word in the name is an abbreviation like DNS :stuck_out_tongue: For example, we have `OpenEdXInstance`, `OpenEdXAppServer`, `DBHandler`, `MySQLServer`, and even `GandiAPI` for the v4 API client. So I would like to keep the naming as-is. What do you say?

Reply:
Keep it as is, it's not really a blocker, just a minor nit. :)